solr_conf/conf/solrconfig.xml

```diff
@@ -35,16 +35,86 @@
       <str name="fl"> *, score</str>
 
       <str name="qf">
-        id
-        descriptive_tiv
+        sw_display_title_tesim^5
+        contributor_text_nostem_im^3
+        topic_tesim^2
+
+        tag_ssim
+
+        originInfo_place_placeTerm_tesim
+        originInfo_publisher_tesim
+
+        content_type_ssim
+        sw_format_ssim
+        object_type_ssim
+
         descriptive_text_nostem_i
+        descriptive_tiv
         descriptive_teiv
-        object_type_ssim
+
+        collection_title_tesim
+
+        id
+        objectId_tesim
+        obj_label_tesim
+        identifier_ssim
+        identifier_tesim
+        barcode_id_ssim
+        folio_instance_hrid_ssim
+        source_id_ssim^3
+        source_id_ssi
+        source_id_text_nostem_i^3
+        previous_ils_ids_ssim
+        doi_ssim
+        contributor_orcids_ssim
       </str>
-      <str name="pf">
-        descriptive_tiv^10
+      <str name="pf">  <!-- (phrase boost within result set) -->
```

Comment:
pf, pf3 and pf2 are about boosting phrase results within the result set ... and we want to do this!  Not sure why it was left off before.